playbooks/hosts-ci

```diff
@@ -0,0 +1,6 @@
+[locals]
```

Comment:
Anyway to auto-generate this rather than having it in the repo? Maybe something for down the road.

Reply:
You should be able to do a big echo in the .yml with all of this info